src/service/url-replacements-impl.js

```diff
@@ -934,6 +937,13 @@ export class UrlReplacements {
    * @private
    */
   expand_(url, opt_bindings, opt_collectVars, opt_sync, opt_whiteList) {
```

Comment:
There are some legacy tests of this that will fail with the new parser. Not sure if its better to refactor tests or code for some of them. @lannka we should discuss next week